entity-framework/core/querying/filters.md

```diff
@@ -36,6 +36,9 @@ The predicate expressions passed to the _HasQueryFilter_ calls will now automati
 > [!TIP]
 > Note the use of a DbContext instance level field: `_tenantId` used to set the current tenant. Model-level filters will use the value from the correct context instance (that is, the instance that is executing the query).
 
+> [!NOTE]
+> It is currently not possible to define multiple query filters on the same entity - only the last one will be applied. However, you can define a single filter with multiple conditions using regular `AND`.
```

Comment:
Nit: `AND` sound very SQL-centered to me. I propose we make it more C#-centric as the query filters experience is based on LINQ. For example:
  
```suggestion
> It is currently not possible to define multiple query filters on the same entity - only the last one will be applied. However, you can define a single filter with multiple conditions using the logical _AND_ operator ([`&&` in C#](https://docs.microsoft.com/dotnet/csharp/language-reference/operators/boolean-logical-operators#conditional-logical-and-operator-)).
```